README.md

```diff
@@ -1 +1,826 @@
-# message-tracing-with-stream-processor
\ No newline at end of file
+[![Build Status](https://travis-ci.org/ballerina-guides/message-tracing-with-stream-processor.svg?branch=master)](https://travis-ci.org/ballerina-guides/message-tracing-with-stream-processor)
+
+# Integration with WSO2 Stream Processor
+
+WSO2 stream processor provides us with distributed message tracing. The Distributed Message Tracer allows you to trace the events which are produced while ballerina services serve for requests. The ballerina services send these tracing data as WSO2 events.
+        
+  >This guide provides instructions on how Ballerina can be used to integrate with Stream-Processor Message Tracer.
+  
+The following are the sections available in this guide.
+
+- [What you'll build](#what-you’ll-build)
+- [Prerequisites](#prerequisites)
+- [Implementation](#implementation)
+- [Testing](#testing)
+- [Testing with distributed message tracing](#testing-with-distributed-message-tracer.)
+     - [Traces](#views-of-traces)
+
+## What you’ll build
+To perform this integration with Distributed Message Tracer, a real world use case of a very simple student management system is used.
+This system will illustrate the manipulation of student details in a school/college management system. The administrator will be able to perform the following actions in this service.
```

Comment:
Let's avoid future tense wherever possible and use present tense. Normally the word will is indicative of future tense. Check for other occurrences of this In this guide.

So you would say, "This system illustrates.." and "The administrator is able..".